### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by ghost** [Gajotres/IonicNativeAudioDemo#1]
Hi, I have tried to check the project but it isn't working. I tried both by running ionic serve, as well as deploying it to an iPhone. When I hit the play button the audio never plays.

**Comment by Gajotres** [comment]
I wrote this example for Android platform; I never tested it on iOS. Have you remotely debugged your application and had you received any error(s)?

**Comment by ghost** [comment]
Yes, I tried both. When I run it I keep getting an error saying nativeaudio is not defined. Can you run it?

**Comment by Gajotres** [comment]
If the plugin is not defined that means you either forget to install it, or installation was corrupted in which case you need to remove the plugin and add it again.

**Comment by rcurvello** [comment]
here too doesn't work

**Comment by stevebissoue** [comment]
lol
